Library/ViewModels/FindFriendsFriendFollowCellViewModel.swift

```diff
@@ -135,6 +143,39 @@ public final class FindFriendsFriendFollowCellViewModel: FindFriendsFriendFollow
       )
       .skipRepeats()
 
+    self.followButtonAccessibilityLabel = name
+      .map {
+        localizedString(
+          key: "Follow_friend_name",
+          defaultValue: "Follow %{friend_name}",
+          count: nil,
+          substitutions: ["friend_name": $0])
+      }
+
+    self.unfollowButtonAccessibilityLabel = name
+      .map {
+        localizedString(
+          key: "Stop_following_friend_name",
+          defaultValue: "Stop following %{friend_name}",
+          count: nil,
+          substitutions: ["friend_name": $0])
+      }
+
+    let followedText = localizedString(
+      key: "Followed",
+      defaultValue: "Followed",
+      count: nil,
+      substitutions: [:])
+
+    let notFollowedText = localizedString(
+      key: "Not_followed",
+      defaultValue: "Not followed",
+      count: nil,
+      substitutions: [:])
+
+    self.cellAccessibilityLabel = Signal.combineLatest(self.name, self.projectsBackedText, isFollowed)
```

Comment:
So this isn't giving the full information for the cell because it's missing the user's location and the created projects information (some users have either or both "backed" and "created" projects). Since you already get all of that for free with VoiceOver, I would suggest just adding an `accessibilityValue` to the cell that sets it to "Followed" or "Not followed"